docs/logs/getting-started-aspnetcore/Program.cs

```diff
@@ -0,0 +1,57 @@
+// <copyright file="Program.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using OpenTelemetry.Logs;
+using OpenTelemetry.Resources;
+
+var builder = WebApplication.CreateBuilder(args);
+
+// Remove default providers and add OpenTelemetry logging provider
+builder.Logging.ClearProviders().AddOpenTelemetry(logging =>
+{
+    logging.IncludeScopes = true;
+
+    var resourceBuilder = ResourceBuilder
+        .CreateDefault()
+        .AddService(builder.Environment.ApplicationName);
+
+    logging.SetResourceBuilder(resourceBuilder)
+        // ConsoleExporter is used for demo purpose only.
+        // In production environment, ConsoleExporter should be replaced with other exporters (e.g. OTLP Exporter).
+        .AddConsoleExporter();
+});
+
+var app = builder.Build();
+
+app.MapGet("/", (ILogger<Program> logger) =>
+{
+    logger.FoodPriceChanged("artichoke", 9.99);
+
+    return "Hello from OpenTelemetry Logs!";
+});
+
+app.Logger.StartingApp();
+
+app.Run();
+
+public static partial class ApplicationLogs
+{
+    [LoggerMessage(EventId = 1, Level = LogLevel.Information, Message = "Starting the app...")]
```

Comment:
What do you think to have a small comment pointing at the link https://learn.microsoft.com/en-us/dotnet/core/extensions/logger-message-generator?

Reply:
nevermind, I am seeing you have some text in the readme about it.